src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -113,6 +120,37 @@ def init_db():
 
     return db_conn
 
+def init_mgmt_interface_tables(db_conn):
+    """
+    Initializes interface maps for mgmt ports
+    :param db_conn: db connector
+    :return: tuple of mgmt name to oid map and mgmt name to alias map
+    """
+
+    db_conn.connect(APPL_DB)
+
+    mgmt_ports_keys = db_conn.keys(APPL_DB, mgmt_if_entry_table(b'*'))
+
+    mgmt_ports = [key.split(mgmt_if_entry_table(b''))[-1]
+        for key in (mgmt_ports_keys if mgmt_ports_keys is not None else [])]
```

Comment:
It's too complex to write into one statement. Could you refactor it more readable?